Week-2/Assignment_2_1/index.html

```diff
@@ -0,0 +1,156 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles/style.css" />
+  </head>
+  <body>
+    <header>
+      <h1>Lorem Ipsum</h1>
+      <h4>
+        "Neque porro quisquam est qui dolorem ipsum quia dolor sit amet,
+        consectetur, adipisci velit..."
+      </h4>
+      <h5>
+        "There is no one who loves pain itself, who seeks after it and wants to
+        have it, simply because it is pain..."
+      </h5>
+    </header>
+    <section id="section">
+      <article>
+        <div>
+          <h2>What is Lorem Ipsum?</h2>
+          <p>
+            <strong>Lorem Ipsum</strong> is simply dummy text of the printing
+            and typesetting industry. Lorem Ipsum has been the industry's
+            standard dummy text ever since the 1500s, when an unknown printer
+            took a galley of type and scrambled it to make a type specimen book.
+            It has survived not only five centuries, but also the leap into
+            electronic typesetting, remaining essentially unchanged. It was
+            popularised in the 1960s with the release of Letraset sheets
+            containing Lorem Ipsum passages, and more recently with desktop
+            publishing software like Aldus PageMaker including versions of Lorem
+            Ipsum.
+          </p>
+        </div>
+      </article>
+      <article>
+        <img src="images/workplace.jpg" height="50%" width="35%" />
```

Comment:
can move the inline styles to external css

Reply:
Removed from html file and added in external css
![image](https://user-images.githubusercontent.com/108617696/182028140-501c603c-bf66-4fb0-a1f7-a72b5cf6f1b0.png)